api-2.2/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ConditionClinicalStatusTranslatorImpl_2_2.java

```diff
@@ -49,6 +49,10 @@ public ConditionClinicalStatus toOpenmrsType(CodeableConcept codeableConcept) {
 	}
 	
 	private ConditionClinicalStatus getClinicalStatus(Coding coding) {
+		if (coding.getCode() == null) {
+			return ConditionClinicalStatus.INACTIVE;
```

Comment:
I would rather assume "ACTIVE"

Reply:
@ibacher 
I set it as INACTIVE because the default value being returned is INACTIVE as well.

Reply:
Ah... ok. That's probably not a great assumption, but fine.